### ISSUE TITLE
Problem to upload image collection in entity

### ISSUE THREAD

**Issue by titoff3** [elcodi/elcodi#1057]
Hello, 

I try to create an entity with images like the product form. It's impossible to get the uploaded files because the field name created in javascript is not a parameter. All the created field has a elcodi_admin_product_form_type_product prefix.

https://github.com/elcodi/bamboo/blob/master/src/Elcodi/Admin/CoreBundle/Resources/public/js/modules/upload-gallery.js#L85

Moreover, for the identifier in the media template, there is a reference to the product too.
https://github.com/elcodi/bamboo/blob/master/src/Elcodi/Admin/MediaBundle/Resources/views/Module/_images-form-field.html.twig#L17

Is it normal, if so, how can I do to make a multiple image field in an other entity.

Thanks.

**Comment by mmoreram** [comment]
@titoff3 you're totally right, we need to change this ASAP.

**Comment by mmoreram** [comment]
ping @tonipinel 

These references to specific fields should be avoided in favor of reusing the images engine. Do you have some time for that?

Thanks.

**Comment by tonipinel** [comment]
Yes! Sure it's pretty easy to change. We need can change the selector to a class and add it to the fields.

**Comment by mmoreram** [comment]
Any update on that @tonipinel ?
